js/tooltip.js

```diff
@@ -357,16 +357,34 @@
 
   }
 
+  Tooltip.prototype.getViewportBounds = function ($viewport) {
+    if ($viewport.is('body') && (/fixed|absolute/).test(this.$element.css('position'))) {
```

Comment:
Possibly due to my C++ background, I'm still amazed with how easy it is to play with strings in js.  @cvrebert do you want me to update it?